apps/polyflix/src/modules/attachments/components/AttachmentAvatar.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { Avatar, IconButton, ListItemIcon, Tooltip } from '@mui/material'
+import CopyToClipboard from 'react-copy-to-clipboard'
+import { useTranslation } from 'react-i18next'
+
+import { useInjection } from '@polyflix/di'
+
+import { Icon } from '@core/components/Icon/Icon.component'
+import { SnackbarService } from '@core/services/snackbar.service'
+
+type Props = {
+  url: string
+  copyToClipboard?: boolean
+}
+
+export const AttachmentAvatar = ({ url, copyToClipboard = true }: Props) => {
+  const { t: tUsers } = useTranslation('users')
+  const { t: tAttachments } = useTranslation('attachments')
```

Comment:
use only one useTranslation('user') or useTranslation('attachements') then if you want to change use namespace

`t('profile.tabs.attachments.content.list.clipboard', { ns: 'user' })`